src/tests/functional/plugin/gpu/single_layer_tests/dynamic/cum_sum.cpp

```diff
@@ -0,0 +1,182 @@
+// Copyright (C) 2018-2023 Intel Corporation
```

Comment:
// Copyright (C) 2023 Intel Corporation

And please rebase and resolve conflicts.

Reply:
fixed